app/views/content/train-to-be-a-teacher/if-you-have-a-degree/_events-near-you.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="events-near-you">
+  <div class="events-near-you__next-event">
+    <div>
+      <h2>
+        <div class="caption">
+          Next event:
+        </div>
+        <br/>
+
+        <div>Train to Teach Middlesborough</div>
+      </h2>
+    </div>
+
+    <div>
+      <div class="date-and-time">
+        <div>29th March 2022</div>
+        <div>16:00 - 20:00</div>
+      </div>
+    </div>
+
+    <div>
+      <a href="/events/question-time-middlesbrough-event-210622" class="button button--white button--inline-block">View event</a>
```

Comment:
There aren't any events on the dev environment that you can point to unfortunately!

Could either point to production (eg https://getintoteaching.education.gov.uk/events/train-to-teach-birmingham-event-260422) or just have no link at all